packages/shared/sierra-client/src/email-verification-notes.ts

```diff
@@ -0,0 +1,128 @@
+import { getVarFieldContent, VarField, varFieldTags } from './patron';
+
+const auth0NotePrefix = 'Auth0:';
+
+// See https://stackoverflow.com/a/9204568
+// Matches string@string.string
+const emailRegex = '\\S+@\\S+\\.\\S+';
```

Comment:
Aargh regex!

I've had a look through all the patron email addresses currently in Sierra. There are a handful that don't match this regex, of which the only one I think we need to care about is https://trustnet.wellcome.org/user/odonnelo.

The others are either clearly nonsense (e.g. `NONE`) or values with trailing/leading whitespace, that we should remove or we won't be able to look them up.

Maybe worth a comment that (1) we know this is an oversimplification and (2) we're okay with it, because we only have a small-ish pool of values.